assets/api.py

```diff
@@ -0,0 +1,29 @@
+# endpoint                            method          result
+#
+# api/assets/                         get             list all assets
+# api/assets/<id>                     get             get a specific asset
+
+
+from rest_framework import serializers, viewsets, status
+from rest_framework.decorators import action
+from rest_framework.response import Response
+from rest_framework.generics import get_object_or_404
+from django.db.models import Min
+from django.core.mail import EmailMessage
+from django.template.loader import get_template
+from django.template import Context, RequestContext
+import datetime
```

Comment:
This is a lot of imports for a small file. I suspect they were copy/pasted, and most can now be removed?

Reply:
They certainly were! `api.py` was not even used, again a leftover. Also sent me looking for more unused imports, they have been also removed